packages/block-editor/README.md

```diff
@@ -422,6 +422,10 @@ _Type_
 
 -   `Object` 
 
+<a name="Text" href="#Text">#</a> **Text**
+
+Renders an editable text input in which text formatting is not allowed.
```

Comment:
I think this might be misinterpreted as an `<input type="text">`.
What about being more candid and go with something like `Renders a contenteditable element in which text formatting is not allowed`?

Reply:
Yes, we need better descriptions. The mobile team previously reminded me not to use `contenteditable` as that's not the case for the native apps. @hypest @SergioEstevao @Tug and maybe others (not sure if @WordPress/native-mobile works), what would be a good description? :)

Reply:
We should also update the `RichText` description. :)

Reply:
Also `Text` might be confusing because in react-native it's [a display-only component](https://facebook.github.io/react-native/docs/text). For this, react-native uses [TextInput](https://facebook.github.io/react-native/docs/textinput.html):

> A foundational component for inputting text into the app via a keyboard. Props provide configurability for several features, such as auto-correction, auto-capitalization, placeholder text, and different keyboard types, such as a numeric keypad

Reply:
Sorry I'm a bit late to the conversation, I'd vote for `TextInput` since it would have very similar behavior as the react-native component @koke talked about.